cli/sendTransactionCmd.go

```diff
@@ -0,0 +1,292 @@
+		for i, key := range ownership.AuthorizedKeys {
+			keyByte, err := hex.DecodeString(key)
+			if err != nil {
+				return nil, err
+			}
+			encrypedSecretKey, err := archethic.EcEncrypt(secretKey, keyByte)
+			if err != nil {
+				return nil, err
+			}
+			authorizedKeysResult[i] = archethic.AuthorizedKey{
+				PublicKey:          keyByte,
+				EncryptedSecretKey: encrypedSecretKey,
+			}
+		}
+		transaction.AddOwnership(cipher, authorizedKeysResult)
+	}
+
+	transaction.SetContent(configuredTransaction.content)
+	transaction.SetCode(configuredTransaction.smartContract)
+
+	return transaction, nil
+}
+
+func mapOwnership(ownerships map[string]string) map[string][]string {
+	result := make(map[string][]string)
+
+	for secret, authorizedKey := range ownerships {
+		if _, ok := result[secret]; !ok {
+			result[secret] = []string{authorizedKey}
+		} else {
+			result[secret] = append(result[secret], authorizedKey)
+		}
+	}
+
+	return result
+}
+
+func GetSendTransactionCmd() *cobra.Command {
+	sendTransactionCmd := &cobra.Command{
+		Use:   "send-transaction",
+		Short: "Send transaction",
+		Run: func(cmd *cobra.Command, args []string) {
+			secretKey := make([]byte, 32)
+			rand.Read(secretKey)
+
+			config, _ := cmd.Flags().GetString("config")
+			var configuredTransaction ConfiguredTransaction
+			var err error
+			if config != "" {
+				configuredTransaction, err = extractTransactionFromInputFile(config)
+				if err != nil {
+					fmt.Println(err)
+					return
+				}
+			} else {
+				configuredTransaction, err = extractTransactionFromInputFlags(cmd)
+				if err != nil {
+					fmt.Println(err)
+					return
+				}
+			}
+
+			client := archethic.NewAPIClient(endpoint.String())
+			storageNouncePublicKey, err := client.GetStorageNoncePublicKey()
+			if err != nil {
+				fmt.Println(err)
+				return
+			}
+
+			txType, err := transactionType.GetTransactionType()
+			if err != nil {
+				fmt.Println(err)
+				return
+			}
+
+			transaction, err := configureTransaction(configuredTransaction, txType, secretKey)
+			if err != nil {
+				fmt.Println(err)
+				return
+			}
+
+			curve, err := ellipticCurve.GetCurve()
+			if err != nil {
+				fmt.Println(err)
+				return
+			}
+
+			serviceMode := configuredTransaction.serviceName != ""
+
+			fmt.Println("Sending transaction...")
+			feedback, err := tuiutils.SendTransaction(*transaction, secretKey, curve, serviceMode, endpoint.String(), configuredTransaction.index, configuredTransaction.serviceName, storageNouncePublicKey, configuredTransaction.accessSeed)
+			if err != nil {
+				fmt.Println(err)
+				return
+			} else {
+				fmt.Println(feedback)
+			}
+		},
+	}
+	sendTransactionCmd.Flags().String("config", "", "The file location of the YAML configuration file")
+	sendTransactionCmd.Flags().Var(&endpoint, "endpoint", "Endpoint (local|testnet|mainnet|[custom url])")
+	sendTransactionCmd.Flags().String("access-seed", "", "Access Seed")
+	sendTransactionCmd.Flags().Int("index", 0, "Index")
```

Comment:
Probably it would be best if by default we fetch the last one, as we don't have a way to get the index from the CLI. Hence, the experience will be better.